packages/angular/cli/lib/config/workspace-schema.json

```diff
@@ -362,6 +362,7 @@
                       "@angular-devkit/build-angular:extract-i18n",
                       "@angular-devkit/build-angular:karma",
                       "@angular-devkit/build-angular:jest",
+                      "@angular-devkit/build-angular:web-test-runner",
```

Comment:
You missing adding the configuration. Example
https://github.com/angular/angular-cli/blob/08b86d25850d43d6fa5c1549f53f9c9dee89712e/packages/angular/cli/lib/config/workspace-schema.json#L544-L565